src/api/app/models/notification/rss_feed_item.rb

```diff
@@ -17,6 +17,37 @@ def self.cleanup
       end
     end
   end
+
+  def self.rss_notification_params(event, subscription)
+    return if subscription.subscriber && subscription.subscriber.away?
+
+    notification_params = {
+      subscriber: subscription.subscriber,
+      event_type: event.eventtype,
+      event_payload: event.payload,
+      subscription_receiver_role: subscription.receiver_role
+    }
+    notification_params.merge!(notification_dynamic_params(event))
+  end
+
+  def self.notification_dynamic_params(event)
```

Comment:
What about moving this into some of the `Event` classes? I feel we're asking too much information about the event here, so It may be better to [tell, don't ask](https://www.martinfowler.com/bliki/TellDontAsk.html)

Reply:
Good advice. Can you have a look again, please?